src/message-list/MessageList.js

```diff
@@ -24,11 +27,13 @@ export default class MessageList extends React.PureComponent {
     return (
       <InfiniteScrollView
         style={styles.list}
+        contentContainerStyle={styles.container}
         automaticallyAdjustContentInset="false"
         autoScrollToBottom={caughtUp}
         stickyHeaderIndices={headerIndices}
         onStartReached={fetchOlder}
         onEndReached={fetchNewer}
+        onScroll={e => {}}
```

Comment:
Because not yet implemented, but not having one throws, so temporarily an empty function.